### ISSUE TITLE
Confirm page doesn't looks good on 107.0b5

### ISSUE THREAD

**Issue by wejick** [mozilla/multi-account-containers#2433]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. Open website that has default container set, outside the default container.
2. The choose container tab will be shown

### Actual behavior

![image](https://i.imgur.com/Y9G8Vt3.png)

### Expected behavior

looks better

### Additional informations

There's this warning on the console
```This page is in Quirks Mode. Page layout may be impacted. For Standards Mode use “<!DOCTYPE html>”.```

### Provide a copy of Troubleshooting Information page (optional)

```plain text
Application Basics
------------------

Name: Firefox
Version: 107.0b5
Build ID: 20221025185841
Distribution ID:
Update Channel: beta
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:107.0) Gecko/20100101 Firefox/107.0
OS: Darwin 21.6.0 Darwin Kernel Version 21.6.0: Thu Sep 29 20:13:46 PDT 2022; root:xnu-8020.240.7~1/RELEASE_ARM64_T8101
Rosetta Translated: false
Multiprocess Windows: 1/1
Fission Windows: 1/1 Enabled by default
Remote Processes: 11
Enterprise Policies: Inactive
Google Location Service Key: Found
Google Safebrowsing Key: Found
Mozilla Location Service Key: Found
Safe Mode: false
Memory Size (RAM): 16.0 GB
Disk Space Available: 353 GB

Crash Reports for the Last 3 Days
---------------------------------

Firefox Features
----------------

Name: Firefox Screenshots
Version: 39.0.1
ID: screenshots@mozilla.org

Name: Form Autofill
Version: 1.0.1
ID: formautofill@mozilla.org

Name: Picture-In-Picture
Version: 1.0.0
ID: pictureinpicture@mozilla.org

Name: Web Compatibility Interventions
Version: 106.0.0
ID: webcompat@mozilla.org

Name: WebCompat Reporter
Version: 1.5.0
ID: webcompat-reporter@mozilla.org

Remote Features
---------------

Remote Processes
----------------

Type: Privileged About
Count: 1

Type: Extension
Count: 1

Type: Isolated Web Content
Count: 5

Type: Preallocated
Count: 3

Type: Socket
Count: 1

Add-ons
-------

Name: English United States Dictionary
Type: dictionary
Version: 91.0
Enabled: true
ID: @unitedstatesenglishdictionary

Name: Add-ons Search Detection
Type: extension
Version: 2.0.0
Enabled: true
ID: addons-search-detection@mozilla.com

Name: Amazon.com
Type: extension
Version: 1.3
Enabled: true
ID: amazondotcom@search.mozilla.org

Name: Bing
Type: extension
Version: 1.3
Enabled: true
ID: bing@search.mozilla.org

Name: ClearURLs
Type: extension
Version: 1.25.0
Enabled: true
ID: {74145f27-f039-47ce-a470-a662b129930a}

Name: Cookie Manager
Type: extension
Version: 1.7
Enabled: true
ID: cookie-manager@robwu.nl

Name: Cookie Remover
Type: extension
Version: 1.0.4
Enabled: true
ID: {3507f56d-2ef5-45c1-b6d7-5297a0ba7642}

Name: Decentraleyes
Type: extension
Version: 2.0.17
Enabled: true
ID: jid1-BoFifL9Vbdl2zQ@jetpack

Name: DuckDuckGo
Type: extension
Version: 1.1
Enabled: true
ID: ddg@search.mozilla.org

Name: Easy Mute for Google Meet
Type: extension
Version: 1.5.2
Enabled: true
ID: {3ca66e29-4853-42e7-8659-200a28565450}

Name: enhanced-h264ify
Type: extension
Version: 2.1.0
Enabled: true
ID: {9a41dee2-b924-4161-a971-7fb35c053a4a}

Name: Facebook Container
Type: extension
Version: 2.3.6
Enabled: true
ID: @contain-facebook

Name: Firefox Multi-Account Containers
Type: extension
Version: 8.0.7
Enabled: true
ID: @testpilot-containers

Name: Google
Type: extension
Version: 1.2
Enabled: true
ID: google@search.mozilla.org

Name: Grammarly: Grammar Checker and Writing App
Type: extension
Version: 8.904.0
Enabled: true
ID: 87677a2c52b84ad3a151a4a72f5bd3c4@jetpack

Name: I don't care about cookies
Type: extension
Version: 3.4.3
Enabled: true
ID: jid1-KKzOGWgsW3Ao4Q@jetpack

Name: JSONView
Type: extension
Version: 2.3.0
Enabled: true
ID: jsonview@brh.numbera.com

Name: Octotree - GitHub code tree
Type: extension
Version: 7.7.0
Enabled: true
ID: jid1-Om7eJGwA1U8Akg@jetpack

Name: Privacy Possum
Type: extension
Version: 2019.7.18
Enabled: true
ID: woop-NoopscooPsnSXQ@jetpack

Name: Refined GitHub
Type: extension
Version: 22.10.12
Enabled: true
ID: {a4c4eda4-fb84-4a84-b4a1-f7c1cbf2a1ad}

Name: slack-redir
Type: extension
Version: 1.3
Enabled: true
ID: {69d7586b-fae9-45a0-a629-3da53bf27b57}

Name: uBlock Origin
Type: extension
Version: 1.44.4
Enabled: true
ID: uBlock0@raymondhill.net

Name: Wappalyzer - Technology profiler
Type: extension
Version: 6.10.41
Enabled: true
ID: wappalyzer@crunchlabz.com

Name: Wikipedia (en)
Type: extension
Version: 1.1
Enabled: true
ID: wikipedia@search.mozilla.org

Name: English (US) Language Pack
Type: locale
Version: 107.0buildid20221025.185841
Enabled: true
ID: langpack-en-US@firefox.mozilla.org

Name: Bookmark Dupes
Type: extension
Version: 6.6
Enabled: false
ID: bookmarkdupes@martin-vaeth.org

Name: Elasticvue
Type: extension
Version: 0.42.0
Enabled: false
ID: {2879bc11-6e9e-4d73-82c9-1ed8b78df296}

Name: Enhancer for YouTube™
Type: extension
Version: 2.0.117.4
Enabled: false
ID: enhancerforyoutube@maximerf.addons.mozilla.org

Name: Greasemonkey
Type: extension
Version: 4.11
Enabled: false
ID: {e4a8a97b-f2ed-450b-b12d-ee082ba24781}

Name: Modify Header Value (HTTP Headers)
Type: extension
Version: 0.1.8
Enabled: false
ID: jid0-oEwF5ZcskGhjFv4Kk4lYc@jetpack

Name: Momentum
Type: extension
Version: 2.6.58
Enabled: false
ID: momentum@momentumdash.com

Name: OneTab
Type: extension
Version: 1.59
Enabled: false
ID: extension@one-tab.com

Name: Reader View
Type: extension
Version: 0.5.3
Enabled: false
ID: {2495d258-41e7-4cd5-bc7d-ac15981f064e}

Name: Tab Reloader (page auto refresh)
Type: extension
Version: 0.5.7
Enabled: false
ID: jid0-bnmfwWw2w2w4e4edvcdDbnMhdVg@jetpack

Name: Tabby - Window and Tab Manager
Type: extension
Version: 3.0.1
Enabled: false
ID: tabby@whatsyouridea.com

Name: Tree Style Tab
Type: extension
Version: 3.9.7
Enabled: false
ID: treestyletab@piro.sakura.ne.jp

Name: User-Agent Switcher and Manager
Type: extension
Version: 0.4.8
Enabled: false
ID: {a6c4a591-f1b2-4f03-b3ff-767e5bedf4e7}

Name: Video DownloadHelper
Type: extension
Version: 7.6.0
Enabled: false
ID: {b9db16a4-6edc-47ec-a1f4-b86292ed211d}

Graphics
--------

Features
Compositing: WebRender
Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled; keyboard enabled; autoscroll enabled; smooth pinch-zoom enabled
WebGL 1 Driver WSI Info: CGL IsWebglOutOfProcessEnabled: 1
WebGL 1 Driver Renderer: Apple -- Apple M1
WebGL 1 Driver Version: 4.1 Metal - 76.3
WebGL 1 Driver Extensions: GL_ARB_blend_func_extended GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_ES2_compatibility GL_ARB_explicit_attrib_location GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader5 GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_occlusion_query2 GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_subroutine GL_ARB_shading_language_include GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_swizzle GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_sRGB_decode GL_APPLE_client_storage GL_APPLE_container_object_shareable GL_APPLE_flush_render GL_APPLE_rgb_422 GL_APPLE_row_bytes GL_APPLE_texture_range GL_NV_texture_barrier
WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_float_blend EXT_frag_depth EXT_shader_texture_lod EXT_sRGB EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_fbo_render_mipmap OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context
WebGL 2 Driver WSI Info: CGL IsWebglOutOfProcessEnabled: 1
WebGL 2 Driver Renderer: Apple -- Apple M1
WebGL 2 Driver Version: 4.1 Metal - 76.3
WebGL 2 Driver Extensions: GL_ARB_blend_func_extended GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_ES2_compatibility GL_ARB_explicit_attrib_location GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader5 GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_occlusion_query2 GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_subroutine GL_ARB_shading_language_include GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_swizzle GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_sRGB_decode GL_APPLE_client_storage GL_APPLE_container_object_shareable GL_APPLE_flush_render GL_APPLE_rgb_422 GL_APPLE_row_bytes GL_APPLE_texture_range GL_NV_texture_barrier
WebGL 2 Extensions: EXT_color_buffer_float EXT_float_blend EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_texture_float_linear WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context
Target Frame Rate: 60
GPU #1
Active: Yes
Vendor ID: 0x106b
RAM: 0

Diagnostics
AzureCanvasBackend: skia
AzureContentBackend: skia
AzureFallbackCanvasBackend: skia
CMSOutputProfile: AAAPwGFwcGwCEAAAbW50clJHQiBYWVogB+YACgAaABAAMQAaYWNzcEFQUEwAAAAAQVBQTAAAAAAAAAAAAAAAAAAAAAAAAPbWAAEAAAAA0y1hcHBsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAARZGVzYwAAAVAAAABiZHNjbQAAAbQAAAScY3BydAAABlAAAAAjd3RwdAAABnQAAAAUclhZWgAABogAAAAUZ1hZWgAABpwAAAAUYlhZWgAABrAAAAAUclRSQwAABsQAAAgMYWFyZwAADtAAAAAgdmNndAAADvAAAAAwbmRpbgAADyAAAAA+bW1vZAAAD2AAAAAodmNncAAAD4gAAAA4YlRSQwAABsQAAAgMZ1RSQwAABsQAAAgMYWFiZwAADtAAAAAgYWFnZwAADtAAAAAgZGVzYwAAAAAAAAAIRGlzcGxheQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAG1sdWMAAAAAAAAAJgAAAAxockhSAAAAFAAAAdhrb0tSAAAADAAAAexuYk5PAAAAEgAAAfhpZAAAAAAAEgAAAgpodUhVAAAAFAAAAhxjc0NaAAAAFgAAAjBkYURLAAAAHAAAAkZubE5MAAAAFgAAAmJmaUZJAAAAEAAAAnhpdElUAAAAGAAAAohlc0VTAAAAFgAAAqByb1JPAAAAEgAAArZmckNBAAAAFgAAAshhcgAAAAAAFAAAAt51a1VBAAAAHAAAAvJoZUlMAAAAFgAAAw56aFRXAAAACgAAAyR2aVZOAAAADgAAAy5za1NLAAAAFgAAAzx6aENOAAAACgAAAyRydVJVAAAAJAAAA1JlbkdCAAAAFAAAA3ZmckZSAAAAFgAAA4ptcwAAAAAAEgAAA6BoaUlOAAAAEgAAA7J0aFRIAAAADAAAA8RjYUVTAAAAGAAAA9BlbkFVAAAAFAAAA3Zlc1hMAAAAEgAAArZkZURFAAAAEAAAA+hlblVTAAAAEgAAA/hwdEJSAAAAGAAABApwbFBMAAAAEgAABCJlbEdSAAAAIgAABDRzdlNFAAAAEAAABFZ0clRSAAAAFAAABGZwdFBUAAAAFgAABHpqYUpQAAAADAAABJAATABDAEQAIAB1ACAAYgBvAGoAac7st+wAIABMAEMARABGAGEAcgBnAGUALQBMAEMARABMAEMARAAgAFcAYQByAG4AYQBTAHoA7QBuAGUAcwAgAEwAQwBEAEIAYQByAGUAdgBuAP0AIABMAEMARABMAEMARAAtAGYAYQByAHYAZQBzAGsA5gByAG0ASwBsAGUAdQByAGUAbgAtAEwAQwBEAFYA5AByAGkALQBMAEMARABMAEMARAAgAGEAIABjAG8AbABvAHIAaQBMAEMARAAgAGEAIABjAG8AbABvAHIATABDAEQAIABjAG8AbABvAHIAQQBDAEwAIABjAG8AdQBsAGUAdQByIA8ATABDAEQAIAZFBkQGSAZGBikEGgQ+BDsETAQ+BEAEPgQyBDgEOQAgAEwAQwBEIA8ATABDAEQAIAXmBdEF4gXVBeAF2V9pgnIATABDAEQATABDAEQAIABNAOAAdQBGAGEAcgBlAGIAbgD9ACAATABDAEQEJgQyBDUEQgQ9BD4EOQAgBBYEGgAtBDQEOARBBD8EOwQ1BDkAQwBvAGwAbwB1AHIAIABMAEMARABMAEMARAAgAGMAbwB1AGwAZQB1AHIAVwBhAHIAbgBhACAATABDAEQJMAkCCRcJQAkoACAATABDAEQATABDAEQAIA4qDjUATABDAEQAIABlAG4AIABjAG8AbABvAHIARgBhAHIAYgAtAEwAQwBEAEMAbwBsAG8AcgAgAEwAQwBEAEwAQwBEACAAQwBvAGwAbwByAGkAZABvAEsAbwBsAG8AcgAgAEwAQwBEA4gDswPHA8EDyQO8A7cAIAO/A7gDzAO9A7cAIABMAEMARABGAOQAcgBnAC0ATABDAEQAUgBlAG4AawBsAGkAIABMAEMARABMAEMARAAgAGEAIABjAG8AcgBlAHMwqzDpMPwATABDAER0ZXh0AAAAAENvcHlyaWdodCBBcHBsZSBJbmMuLCAyMDIyAABYWVogAAAAAAAA81EAAQAAAAEWzFhZWiAAAAAAAACD3wAAPb////+7WFlaIAAAAAAAAEq/AACxNwAACrlYWVogAAAAAAAAKDgAABELAADIuWN1cnYAAAAAAAAEAAAAAAUACgAPABQAGQAeACMAKAAtADIANgA7AEAARQBKAE8AVABZAF4AYwBoAG0AcgB3AHwAgQCGAIsAkACVAJoAnwCjAKgArQCyALcAvADBAMYAywDQANUA2wDgAOUA6wDwAPYA+wEBAQcBDQETARkBHwElASsBMgE4AT4BRQFMAVIBWQFgAWcBbgF1AXwBgwGLAZIBmgGhAakBsQG5AcEByQHRAdkB4QHpAfIB+gIDAgwCFAIdAiYCLwI4AkECSwJUAl0CZwJxAnoChAKOApgCogKsArYCwQLLAtUC4ALrAvUDAAMLAxYDIQMtAzgDQwNPA1oDZgNyA34DigOWA6IDrgO6A8cD0wPgA+wD+QQGBBMEIAQtBDsESARVBGMEcQR+BIwEmgSoBLYExATTBOEE8AT+BQ0FHAUrBToFSQVYBWcFdwWGBZYFpgW1BcUF1QXlBfYGBgYWBicGNwZIBlkGagZ7BowGnQavBsAG0QbjBvUHBwcZBysHPQdPB2EHdAeGB5kHrAe/B9IH5Qf4CAsIHwgyCEYIWghuCIIIlgiqCL4I0gjnCPsJEAklCToJTwlkCXkJjwmkCboJzwnlCfsKEQonCj0KVApqCoEKmAquCsUK3ArzCwsLIgs5C1ELaQuAC5gLsAvIC+EL+QwSDCoMQwxcDHUMjgynDMAM2QzzDQ0NJg1ADVoNdA2ODakNww3eDfgOEw4uDkkOZA5/DpsOtg7SDu4PCQ8lD0EPXg96D5YPsw/PD+wQCRAmEEMQYRB+EJsQuRDXEPURExExEU8RbRGMEaoRyRHoEgcSJhJFEmQShBKjEsMS4xMDEyMTQxNjE4MTpBPFE+UUBhQnFEkUahSLFK0UzhTwFRIVNBVWFXgVmxW9FeAWAxYmFkkWbBaPFrIW1hb6Fx0XQRdlF4kXrhfSF/cYGxhAGGUYihivGNUY+hkgGUUZaxmRGbcZ3RoEGioaURp3Gp4axRrsGxQbOxtjG4obshvaHAIcKhxSHHscoxzMHPUdHh1HHXAdmR3DHeweFh5AHmoelB6+HukfEx8+H2kflB+/H+ogFSBBIGwgmCDEIPAhHCFIIXUhoSHOIfsiJyJVIoIiryLdIwojOCNmI5QjwiPwJB8kTSR8JKsk2iUJJTglaCWXJccl9yYnJlcmhya3JugnGCdJJ3onqyfcKA0oPyhxKKIo1CkGKTgpaymdKdAqAio1KmgqmyrPKwIrNitpK50r0SwFLDksbiyiLNctDC1BLXYtqy3hLhYuTC6CLrcu7i8kL1ovkS/HL/4wNTBsMKQw2zESMUoxgjG6MfIyKjJjMpsy1DMNM0YzfzO4M/E0KzRlNJ402DUTNU01hzXCNf02NzZyNq426TckN2A3nDfXOBQ4UDiMOMg5BTlCOX85vDn5OjY6dDqyOu87LTtrO6o76DwnPGU8pDzjPSI9YT2hPeA+ID5gPqA+4D8hP2E/oj/iQCNAZECmQOdBKUFqQaxB7kIwQnJCtUL3QzpDfUPARANER0SKRM5FEkVVRZpF3kYiRmdGq0bwRzVHe0fASAVIS0iRSNdJHUljSalJ8Eo3Sn1KxEsMS1NLmkviTCpMcky6TQJNSk2TTdxOJU5uTrdPAE9JT5NP3VAnUHFQu1EGUVBRm1HmUjFSfFLHUxNTX1OqU/ZUQlSPVNtVKFV1VcJWD1ZcVqlW91dEV5JX4FgvWH1Yy1kaWWlZuFoHWlZaplr1W0VblVvlXDVchlzWXSddeF3JXhpebF69Xw9fYV+zYAVgV2CqYPxhT2GiYfViSWKcYvBjQ2OXY+tkQGSUZOllPWWSZedmPWaSZuhnPWeTZ+loP2iWaOxpQ2maafFqSGqfavdrT2una/9sV2yvbQhtYG25bhJua27Ebx5veG/RcCtwhnDgcTpxlXHwcktypnMBc11zuHQUdHB0zHUodYV14XY+dpt2+HdWd7N4EXhueMx5KnmJeed6RnqlewR7Y3vCfCF8gXzhfUF9oX4BfmJ+wn8jf4R/5YBHgKiBCoFrgc2CMIKSgvSDV4O6hB2EgITjhUeFq4YOhnKG14c7h5+IBIhpiM6JM4mZif6KZIrKizCLlov8jGOMyo0xjZiN/45mjs6PNo+ekAaQbpDWkT+RqJIRknqS45NNk7aUIJSKlPSVX5XJljSWn5cKl3WX4JhMmLiZJJmQmfyaaJrVm0Kbr5wcnImc951kndKeQJ6unx2fi5/6oGmg2KFHobaiJqKWowajdqPmpFakx6U4pammGqaLpv2nbqfgqFKoxKk3qamqHKqPqwKrdavprFys0K1ErbiuLa6hrxavi7AAsHWw6rFgsdayS7LCszizrrQltJy1E7WKtgG2ebbwt2i34LhZuNG5SrnCuju6tbsuu6e8IbybvRW9j74KvoS+/796v/XAcMDswWfB48JfwtvDWMPUxFHEzsVLxcjGRsbDx0HHv8g9yLzJOsm5yjjKt8s2y7bMNcy1zTXNtc42zrbPN8+40DnQutE80b7SP9LB00TTxtRJ1MvVTtXR1lXW2Ndc1+DYZNjo2WzZ8dp22vvbgNwF3IrdEN2W3hzeot8p36/gNuC94UThzOJT4tvjY+Pr5HPk/OWE5g3mlucf56noMui86Ubp0Opb6uXrcOv77IbtEe2c7ijutO9A78zwWPDl8XLx//KM8xnzp/Q09ML1UPXe9m32+/eK+Bn4qPk4+cf6V/rn+3f8B/yY/Sn9uv5L/tz/bf//cGFyYQAAAAAAAwAAAAJmZgAA8qcAAA1ZAAAT0AAAClt2Y2d0AAAAAAAAAAEAAQAAAAAAAAABAAAAAQAAAAAAAAABAAAAAQAAAAAAAAABAABuZGluAAAAAAAAADYAAK4UAABR7AAAQ9cAALCkAAAmZgAAD1wAAFANAABUOQACMzMAAjMzAAIzMwAAAAAAAAAAbW1vZAAAAAAAAAYQAACgSf1ibWIAAAAAAAAAAAAAAAAAAAAAAAAAAHZjZ3AAAAAAAAMAAAACZmYAAwAAAAJmZgADAAAAAmZmAAAAAjMzNAAAAAACMzM0AAAAAAIzMzQA
Display0: 2880x1800@0Hz scales:2.000000|2.000000
DisplayCount: 1
Decision Log
HW_COMPOSITING:
available by default
OPENGL_COMPOSITING:
available by default
WEBRENDER:
available by default
WEBRENDER_QUALIFIED:
available by default
WEBRENDER_COMPOSITOR:
available by default
WEBRENDER_PARTIAL:
disabled by default: User disabled via pref
WEBRENDER_SHADER_CACHE:
disabled by default: Disabled by default
WEBRENDER_OPTIMIZED_SHADERS:
available by default
WEBRENDER_ANGLE:
available by default
unavailable by env: OS not supported
WEBRENDER_DCOMP_PRESENT:
available by default
disabled by user: User disabled via pref
unavailable by env: Requires Windows 10 or later
unavailable by runtime: Requires ANGLE
WEBRENDER_SOFTWARE:
available by default
WEBGPU:
disabled by default: Disabled by default
blocked by runtime: WebGPU cannot be enabled in release or beta
VP8_HW_DECODE:
available by default
VP9_HW_DECODE:
available by default
BACKDROP_FILTER:
available by default




Media
-----

Audio Backend: audiounit-rust
Max Channels: 2
Preferred Sample Rate: 48000
Roundtrip latency (standard deviation): ...
Output Devices
Name: Group
ZoomAudioDevice: zoom.us.zoomaudiodevice
MacBook Pro Speakers: builtin-internal-mic|spk
Input Devices
Name: Group
ZoomAudioDevice: zoom.us.zoomaudiodevice
MacBook Pro Microphone: builtin-internal-mic|spk

Enumerate database

Environment Variables
---------------------

MOZ_CRASHREPORTER_EVENTS_DIRECTORY: /Users/-/Library/Application Support/Firefox/Profiles/n7vsmyfg.default-beta/crashes/events
MOZ_CRASHREPORTER_RESTART_ARG_0: /Applications/Firefox.app/Contents/MacOS/firefox
MOZ_CRASHREPORTER_RESTART_ARG_1:
MOZ_CRASHREPORTER_DATA_DIRECTORY: /Users/-/Library/Application Support/Firefox/Crash Reports
MOZ_CRASHREPORTER_PING_DIRECTORY: /Users/-/Library/Application Support/Firefox/Pending Pings
MOZ_CRASHREPORTER_STRINGS_OVERRIDE: /Applications/Firefox.app/Contents/Resources/browser/crashreporter-override.ini
MOZ_LAUNCHED_CHILD:
MOZ_APP_SILENT_START:
XRE_PROFILE_PATH:
XRE_PROFILE_LOCAL_PATH:
XRE_START_OFFLINE:
XRE_BINARY_PATH:
XRE_RESTARTED_BY_PROFILE_MANAGER:

Experimental Features
---------------------

about:home startup cache (browser.startup.homepage.abouthome_cache.enabled): false
Accessibility cache (accessibility.cache.enabled): false
Cookies: SameSite=Lax by default (network.cookie.sameSite.laxByDefault): true
Cookies: SameSite=None requires secure attribute (network.cookie.sameSite.noneRequiresSecure): true
CSS: Masonry Layout (layout.css.grid-template-masonry-value.enabled): false
Developer Tools: Compatibility Panel (devtools.inspector.compatibility.enabled): true
Developer Tools: Service Worker debugging (devtools.debugger.features.windowless-service-workers): false
Media: JPEG XL (image.jxl.enabled): false
Address Bar: show results during IME composition (browser.urlbar.keepPanelOpenDuringImeComposition): false
Web API: WebGPU (dom.webgpu.enabled): false
WebRTC Global Mute Toggles (privacy.webrtc.globalMuteToggles): false

Remote Experiments
------------------

Task Continuity - Sync after tab change Rollout 40%: sync-after-tab
TCP rollout, Beta Phase III (TCP on by default for remaining Beta profiles): control

Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.contentblocking.category: custom
browser.search.region: ID
browser.sessionstore.upgradeBackup.latestBuildID: 20221025185841
browser.startup.homepage_override.buildID: 20221025185841
browser.startup.homepage_override.mstone: 107.0
browser.startup.page: 3
browser.tabs.firefox-view.ui-state.recently-closed-tabs.open: false
browser.tabs.firefox-view.ui-state.tab-pickup.open: false
browser.urlbar.maxRichResults: 12
browser.urlbar.placeholderName: Google
browser.urlbar.placeholderName.private: Google
browser.urlbar.quicksuggest.migrationVersion: 2
browser.urlbar.quicksuggest.scenario: history
browser.urlbar.resultGroups: {"children":[{"maxResultCount":1,"children":[{"group":"heuristicTest"},{"group":"heuristicExtension"},{"group":"heuristi
browser.urlbar.showSearchSuggestionsFirst: false
browser.urlbar.suggest.calculator: true
browser.urlbar.tipShownCount.searchTip_onboard: 4
browser.urlbar.tipShownCount.searchTip_redirect: 4
doh-rollout.balrog-migration-done: true
doh-rollout.disable-heuristics: true
doh-rollout.doneFirstRun: true
doh-rollout.home-region: ID
dom.security.https_only_mode_ever_enabled: true
extensions.lastAppVersion: 107.0
general.autoScroll: false
idle.lastDailyNotification: 1666754347
media.eme.enabled: false
media.gmp-gmpopenh264.abi: aarch64-gcc3
media.gmp-gmpopenh264.lastDownload: 1661694028
media.gmp-gmpopenh264.lastInstallStart: 1661694027
media.gmp-gmpopenh264.lastUpdate: 1661694028
media.gmp-gmpopenh264.version: 1.8.1.2
media.gmp-manager.buildID: 20221025185841
media.gmp-manager.lastCheck: 1666754073
media.gmp-manager.lastEmptyCheck: 1666754073
media.gmp.storage.version.observed: 1
media.videocontrols.picture-in-picture.video-toggle.has-used: true
network.dns.disablePrefetch: true
network.http.speculative-parallel-limit: 0
network.predictor.enabled: false
network.prefetch-next: false
network.trr.mode: 2
network.trr.uri: https://firefox.dns.nextdns.io/
places.database.lastMaintenance: 1666347128
privacy.donottrackheader.enabled: true
privacy.purge_trackers.date_in_cookie_database: 0
privacy.purge_trackers.last_purge: 1666754347765
privacy.restrict3rdpartystorage.rollout.enabledByDefault: true
privacy.sanitize.pending: [{"id":"newtab-container","itemsToClear":[],"options":{}}]
privacy.userContext.enabled: true
privacy.userContext.extension: @testpilot-containers
privacy.userContext.newTabContainerOnLeftClick.enabled: true
privacy.userContext.ui.enabled: true
security.default_personal_cert: Select Automatically
security.remote_settings.crlite_filters.checked: 1654136833
security.remote_settings.intermediates.checked: 1653967226
security.sandbox.content.tempDirSuffix: 62df9aa7-8a9e-46a1-87c9-877a5c8317a1
services.sync.declinedEngines:
services.sync.engine.bookmarks.validation.lastTime: 1666754008
services.sync.engine.creditcards: true
services.sync.engine.passwords.validation.lastTime: 1666754008
services.sync.engine.prefs.modified: false
services.sync.lastPing: 1666754005
services.sync.lastSync: Wed Oct 26 2022 21:11:20 GMT+0700 (Western Indonesia Time)
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1664767275
svg.context-properties.content.enabled: true

Important Locked Preferences
----------------------------

fission.autostart.session: true

Places Database
---------------

Accessibility
-------------

Activated: true
Prevent Accessibility: 0

Library Versions
----------------

NSPR
Expected minimum version: 4.35
Version in use: 4.35

NSS
Expected minimum version: 3.84
Version in use: 3.84

NSSSMIME
Expected minimum version: 3.84
Version in use: 3.84

NSSSSL
Expected minimum version: 3.84
Version in use: 3.84

NSSUTIL
Expected minimum version: 3.84
Version in use: 3.84

Sandbox
-------

Content Process Sandbox Level: 3
Effective Content Process Sandbox Level: 3
Win32k Lockdown State for Content Process: Win32k Lockdown disabled -- Operating system not supported

Startup Cache
-------------

Disk Cache Path: /Users/gian.giovani/Library/Caches/Firefox/Profiles/n7vsmyfg.default-beta/startupCache/startupCache.8.little
Ignore Disk Cache: false
Found Disk Cache on Init: true
Wrote to Disk Cache: false

Internationalization & Localization
-----------------------------------

Application Settings
Requested Locales: ["en-US","id"]
Available Locales: ["id","en-US"]
App Locales: ["en-US","id"]
Regional Preferences: ["en-ID"]
Default Locale: "id"
Operating System
System Locales: ["en-ID"]
Regional Preferences: ["en-ID"]

Remote Debugging (Chromium Protocol)
------------------------------------

Accepting Connections: false
URL:

Printing
--------

Modified print settings
-----------------------

print_printer: Mozilla Save to PDF
```

**Comment by rickdoesburg** [comment]
Having the same issue for a couple of weeks now. Thought I broke something but both my Macs have the same issue. I'm on 108.0a1

**Comment by dannycolin** [comment]
duplicate of #2423
